infra/terraform/global/modules/lambda_mgmt/CHANGELOG.md

```diff
@@ -1,9 +0,0 @@
-# Changelog
```

Comment:
How is it that the Changelog got dumped? Wasn't it up-to-date? I still find changelogs very useful and quicker than checking the git commits but maybe I'm missing something specific of this module.

Reply:
The CHANGELOG for the module was pretty much empty and wrong. 

- We don't version the module
- We can't version the module